src/link-utils.js

```diff
@@ -0,0 +1,35 @@
+const parseRawLinks = (rawLinks) => {
+  if (typeof rawLinks !== "string") {
+    return [];
+  }
+
+  let links = [];
+  const splitLinks = rawLinks.split(",");
+
+  for (var i = 0; i < splitLinks.length; i++) {
+    let link = {};
+    const splitParts = splitLinks[i].split(";");
+
+    link.href = splitParts[0].replace(/<(.*)>/, "$1").trim();
+    link.rel = splitParts[1].replace(/rel="(.*)"/, "$1").trim();
+    link.title = splitParts[2].replace(/title="(.*)"/, "$1").trim();
+
+    links.push(link);
+  }
+
+  return links;
+};
+
+const resolveUrls = (links, baseUrl) => {
+  const linksCopy = Object.assign([], links);
```

Comment:
I don't think you should be using Object.assign with an array. it's used for objects.. weird how it works for arrays too. I think you're looking for array.splice maybe?

Reply:
I got it to work using `array.slice(0)`